### DISCUSSION TITLE
Disabling Modal scale animation

### DISCUSSION THREAD

**Discussion by enzo-billis** [chakra-ui/chakra-ui#8404] [Q&A]
Hi !
I've search for hours but not found any acceptable answers. I would like to disable scale modal animation and keep the opacity animation.
I know the prop motionPreset="none" exists but it's disable scale AND opacity preset. I tried some hack like scale="none" on ModalContent but it doesn't seems to work.
Do you have any ideas ? Should I create a feature request for this ?
Thanks

**Comment by segunadebayo** [comment]
Modal is `Dialog` now. Control motion on `Dialog.Content`.
